pages/page.py

```diff
@@ -80,8 +78,6 @@ def wait_for_element_present(self, *locator):
         self.selenium.implicitly_wait(0)
         try:
             WebDriverWait(self.selenium, 10).until(lambda s: self._selenium_root.find_element(*locator))
-        except TimeoutException:
-            Assert.fail(TimeoutException)
```

Comment:
This is weird. Let's also see if we can remove implicit waits after migrating to pytest-selenium. :smile: